src/elements/content-explorer/Content.js

```diff
@@ -44,67 +48,46 @@ type Props = {
     onSortChange: Function,
     rootElement?: HTMLElement,
     rootId: string,
+    slotRenderer?: Function,
     tableRef: Function,
     view: View,
+    viewMode?: ViewMode,
 };
 
 const Content = ({
-    view,
-    isSmall,
-    isMedium,
-    isTouch,
-    rootId,
-    rootElement,
+    columnCount = 1,
     currentCollection,
-    tableRef,
-    focusedRow,
-    canDownload,
-    canDelete,
-    canRename,
-    canShare,
-    canPreview,
-    onItemClick,
-    onItemSelect,
-    onItemDelete,
-    onItemDownload,
-    onItemRename,
-    onItemShare,
-    onItemPreview,
-    onSortChange,
-}: Props) => (
-    <div className="bce-content">
-        {view === VIEW_ERROR || view === VIEW_SELECTED ? null : (
-            <ProgressBar percent={currentCollection.percentLoaded} />
-        )}
-        {isEmpty(view, currentCollection) ? (
-            <EmptyState view={view} isLoading={currentCollection.percentLoaded !== 100} />
-        ) : (
-            <ItemList
-                view={view}
-                isSmall={isSmall}
-                isMedium={isMedium}
-                isTouch={isTouch}
-                rootId={rootId}
-                rootElement={rootElement}
-                focusedRow={focusedRow}
-                currentCollection={currentCollection}
-                tableRef={tableRef}
-                canShare={canShare}
-                canPreview={canPreview}
-                canDelete={canDelete}
-                canRename={canRename}
-                canDownload={canDownload}
-                onItemClick={onItemClick}
-                onItemSelect={onItemSelect}
-                onItemDelete={onItemDelete}
-                onItemDownload={onItemDownload}
-                onItemRename={onItemRename}
-                onItemShare={onItemShare}
-                onItemPreview={onItemPreview}
-                onSortChange={onSortChange}
-            />
-        )}
-    </div>
-);
+    slotRenderer = index => <div>{index}</div>,
```

Comment:
Is this a placeholder or the real default for the `slotRenderer` of content explorer?

Reply:
I was just using that as a placeholder so something would render to the screen for this PR.  That default will not be there in the final version of the code.  Is that ok to keep, or should I remove it?